app/panel/components/Summary.jsx

```diff
@@ -686,6 +696,29 @@ class Summary extends React.Component {
 		);
 	}
 
+	_renderRewardsIcon() {
```

Comment:
Can we add a quick comment header on this method?

Reply:
Done